experimental/rules_python/wheelmaker.py

```diff
@@ -101,6 +101,11 @@ def arcname_from(name):
 
             return normalized_arcname
 
+        # If anywhere in the dependency tree a directory is set
+        # it will be passed in as an input file and we need to
+        # ignore.
+        if os.path.isdir(real_filename):
+            return
         arcname = arcname_from(package_filename)
 
         self._zipfile.write(real_filename, arcname=arcname)
```

Comment:
Double checking that we actually don't want to write the directory as a file?  That seems right.

Reply:
I don't think we do as we would likely be creating the directory in another test with files in it. I am not sure people would want an empty directory placed into the wheel but have been suprised before by what people do.